src/io/flutter/view/HighlightedTable.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.view;
+
+import com.intellij.ui.JBColor;
+import com.intellij.ui.table.JBTable;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Font;
+import java.awt.event.MouseEvent;
+import javax.swing.event.MouseInputAdapter;
+import javax.swing.table.TableCellRenderer;
+import javax.swing.table.TableModel;
+
+/**
+ * JBTable custom selection and hover renderer for a table view.
+ */
+public class HighlightedTable extends JBTable {
+  final private JBColor HOVER_BACKGROUND_COLOR =
+    new JBColor(new Color(0xCFE6EF), JBColor.LIGHT_GRAY.brighter());
+  final private JBColor HOVER_FOREGROUND_COLOR = JBColor.BLACK;
```

Comment:
drive by comment: I think you meant to make these color constants static.

Reply:
Ooops.